pkg/image/apis/image/helper.go

```diff
@@ -41,6 +46,16 @@ const (
 	ImportRegistryNotAllowed = "registry is not allowed for import"
 )
 
+// knownSignatureStores is a list of known signature stores. The key represents
+// a registry name and it maps to a signature store for that registry. The
+// signature store has to be HTTPS and it must offer image signatures based on
+// this format: https://STORE_URL/IMAGE_NAME/signature-1
+// The response must be valid image signature blob.
+// TODO: Maybe this should be configurable via master config?
```

Comment:
(Not really reviewing right now, will take a look later.)

Shouldn’t this just use the host’s https://github.com/containers/image/blob/master/docs/registries.d.md ? Or, ultimately, just leave the fetching and configuration parsing to the`github.com/containers/image/docker` client?

Is it inherently necessary, or desirable at all, for the cluster to use a _separate_ configuration from the hosts it is running on?